### DISCUSSION TITLE
Welcome to the GetSimpleCMS CE Forum!

### DISCUSSION THREAD

**Discussion by GetSimpleCMS-CE** [GetSimpleCMS-CE/GetSimpleCMS-CE#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by FL-Retiree** [comment]
I've long awaited the inclusion of a built-in feature or a plugin that would allow for easy inclusion of external links in navigation menus and the option to include target="_blank" in the link.  Other CMS offerings like Wordpress and Joomla provide these capabilities.  It seems that many users of the original GetSimple wished for this capability also.  I am so excited to see that this CE edition of GetSimple is evolving as it is.  Thanks much.

**Reply by risingisland** [reply]
Glad to hear that someone is finding these updates useful. More great thing coming in the near future.
Also, thanks for being our first commenter on the new forum!

**Comment by fahad4x4** [comment]
Thank you from the bottom of my heart. You saved my site from collapsing, so thank you.

**Comment by mar3k2004** [comment]
Great job!
It's nice that the project is still alive and in my opinion much better ;-)

**Comment by risingisland** [comment]
@fahad4x4 @mar3k2004 and to everyone else, we are glad to here that everyone is enjoying the new & improved version.
Keep your eyes open, in the coming weeks we hope to be releasing the next version. Furthering the security and features  of GS-CE.
We have more plans in store too. Expanding the Wiki for more plugin info, adding YouTube tutorials, new plugins and themes.

And dont be afraid to [buy us a coffee](https://getsimple-ce.ovh/donate) to keep us awake and coding, if you are finding this useful for you or your clients. A small gesture goes a long ways.

**Comment by barthusz** [comment]
Really happy to see that you guys keep this great CMS alive!

**Comment by tikita** [comment]
Hello,

was searching in the _GetSimpleCMS-CE wiki_, but: how can I change the **logo** in the header?

After installation (GetSimple CE – 3.3.22.) it is <https://avatars.githubusercontent.com/u/119761508?v=4> with /theme/ResponsiveCE/.

Thank you, Tikita

**Comment by risingisland** [comment]
Hello Tikita.

In the admin, go to tab: `Theme > Edit Theme` 
Choose file: `header.inc.php`
And edit line 31
![image](https://github.com/user-attachments/assets/b5a015c2-d46d-46ba-a420-2b73b12f7784)

Change to:
`<img src="<?php get_theme_url(); ?>images/my-new-logo.png" style="width:60px;">`
if you new logo is in the themes folder.
or
`<img src="<?php get_data_uploads(); ?>my-new-logo.png" style="width:60px;">`
if you new logo is in the uploads folder.

**Reply by tikita** [reply]
Wow, thank you so much risingisland for your quick help!

It worked with your instructions,

best regards Tikita
